build.gradle

```diff
@@ -164,13 +165,15 @@ dependencies {
     //database
     compile("com.h2database:h2:$h2_version")
     compile("mysql:mysql-connector-java:$mysql_version")
+    compile("org.mariadb.jdbc:mariadb-java-client:$mariadb_version")
```

Comment:
What is this for?  Just curious..

Reply:
Ahh good eyes.  Forgot to point this one out.  I came across it while researching what the JDBC string should be for RDS/Aurora.  It has better built in support for Aurora's failover mechanisms.  So thinking of using that instead.  Fun fact I also learned, MariaDB is a fork of MySql after Oracle took over the latter and is run by the original contributors to MySql